lib/actions/model.js

```diff
@@ -45,8 +46,18 @@ module.exports = {
       if (preSaveAction) dispatch(preSaveAction);
 
       const Model = storageManager.storage(brainstemKey).model;
-      const xhr = new Model({ id: modelId })
-        .save(attributes, saveOptions);
+      const model = new Model({ id: modelId });
+
+      if (typeof model.validate === 'function') {
+        const validationErrors = model.validate(attributes);
+
+        if (validationErrors) {
```

Comment:
It might be better to just check whether the model [`isValid`](http://backbonejs.org/#Model-isValid)